framework/framework_test.go

```diff
@@ -1,316 +0,0 @@
-package framework
-
-import (
-	"net"
-	"reflect"
-	"sync"
-	"testing"
-
-	"github.com/coreos/go-etcd/etcd"
-	"github.com/taskgraph/taskgraph"
-	"github.com/taskgraph/taskgraph/controller"
-	"github.com/taskgraph/taskgraph/example/topo"
-	"github.com/taskgraph/taskgraph/framework/frameworkhttp"
-	"github.com/taskgraph/taskgraph/pkg/etcdutil"
-	"golang.org/x/net/context"
-)
-
-// TestRequestDataEpochMismatch creates a scenario where data request happened
-// with two different epochs. In this case, the server should back pressure and
-// request client should get notified and return error.
-func TestRequestDataEpochMismatch(t *testing.T) {
```

Comment:
Will add all test back and
1. change TestRequestDataEpochMismatch to TestFetchErrEpochChanged.
2. change TestFrameworkDataRequest to TestFrameworkFetch